README.md

```diff
@@ -14,6 +14,16 @@ The SOC Management widget provides an overall view of an investigation scenario
 
 **Applicable**: Dashboards
 
+
+## Release Notes
+
+Following enhancements have been made to the SOC Management Widget in version 1.0.1:
+
+- Fixed bug: Incorrect "Percentage change" value for ROI, Actions Executed and Overall Time Saved
```

Comment:
This could be written as, "Percentage change" value for ROI, Actions Executed and Overall Time Saved is now shown correctly incase there is no data available. What do you think?